### ISSUE TITLE
[HOLD] [$250] Web - Attachment- Send button located through the entire Expensify app, covering LHN page

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#8594]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
Pre-condition: have you PDF file set with protected password

1. Go to https://staging.new.expensify.com
2. Log in with any account  and select any chat
3. Click on plus button
4. Select add attachment > select Document.
5. Choose any password protected PDF.

## Expected Result:
Send button should displayed in chat area

## Actual Result:
Send button located through the entire Expensify app, covering LHN page

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- Web

**Version Number:** 1.1.54

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**Email or phone of affected tester (no customers):** applausetester+0901abb@applause.expensifail.com

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

![Web issue](https://user-images.githubusercontent.com/93399543/162759588-89fc2748-94b5-40ce-8f2c-dd963982d68d.png)

**Expensify/Expensify Issue URL:** 

**Issue reported by:** Applause

**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by Christinadobrzyn** [comment]
Posted job in Upwork: 

Internal posting - https://www.upwork.com/ab/applicants/1514409181685161984/job-details
External posting - https://www.upwork.com/jobs/~0179e1c06122700be7

Waiting on proposals!

**Comment by Christinadobrzyn** [comment]
I'm going to be ooo until April 26th so I'm going to assign a new CM to this issue.

**Comment by adelekennedy** [comment]
[internal](https://www.upwork.com/ab/applicants/1514684595933376512/job-details)
[external](https://www.upwork.com/jobs/~01f4af78f09c7085ee)

**Comment by adelekennedy** [comment]
still waiting on proposals! It looks like we still have a day or two until we're doubling

**Comment by parasharrajat** [comment]
This is just a UI glitch due to the password prompt. There is another issue which is changing the prompt UI so this issue will not exist after that is done. https://expensify.slack.com/archives/C02NK2DQWUX/p1649968961380469

**Comment by Santhosh-Sellavel** [comment]
Thanks, @parasharrajat!. 

@adelekennedy Can you add HOLD to this issue, I'll follow up on this once https://github.com/Expensify/App/issues/8595 PR is up.

**Comment by stitesExpensify** [comment]
Not overdue, on hold

**Comment by stitesExpensify** [comment]
Still on hold

**Comment by stitesExpensify** [comment]
Not overdue, on hold

**Comment by adelekennedy** [comment]
Still on hold

**Comment by adelekennedy** [comment]
are we still holding in this? @stitesExpensify @Santhosh-Sellavel?

**Comment by Santhosh-Sellavel** [comment]
Yes on hold!

**Comment by stitesExpensify** [comment]
holding

**Comment by adelekennedy** [comment]
on hold still

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by stitesExpensify** [comment]
Not overdue, issue was not reproducible last time it was tested

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (Second week)

**Comment by adelekennedy** [comment]
So in this case the issue isn't being reproduced but it's still on hold - should we close this? @stitesExpensify @Santhosh-Sellavel?

**Comment by stitesExpensify** [comment]
Yep! Let's just close it out.
